.github/workflows/tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
It seems to be the default, and since we’re running everything in a container inside it anyway (with a similarly specific image python:2.7-alpine3.11, matching production), I think it’s okay.